.changeset/shaggy-singers-promise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ai-sdk/provider-utils': major
```

Comment:
change to patch

Reply:
fixed at [8c7fcc7](https://github.com/vercel/ai/pull/6197/commits/8c7fcc7d14715f0e9b12117fd53b9d002aae898a)